src/models/BusinessesModel.ts

```diff
@@ -33,9 +33,7 @@ const BusinessesModel = types
           ),
         );
         if (response.stat === 'ok') {
```

Comment:
Do we need this check now?

Reply:
Yes it is expecting this check as the data is coming only on the success case.

Reply:
But if only success case comes here then stat is always 'ok' then why check this here?